src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/controller/v1/FlowController.java

```diff
@@ -137,7 +137,7 @@ public CompletableFuture<FlowResponsePayload> deleteFlow(@PathVariable(name = "f
     @ResponseStatus(HttpStatus.OK)
     public CompletableFuture<FlowResponsePayload> updateFlow(@PathVariable(name = "flow-id") String flowId,
                                                              @RequestBody FlowUpdatePayload flow) {
-        return flowService.updateFlow(flow);
+        return flowService.updateFlow(flowId, flow);
```

Comment:
Flow Id is also present in the payload and it could differ from the path variable? I am wondering why don't we put the path variable inside the payload? Possibly raise an error if they are not the same

Reply:
This was discussed in the issue https://github.com/telstra/open-kilda/issues/5075#issuecomment-1422517669